src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,32 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.PasswordRepeated;
+
+@PasswordRepeated
 public class UserRequestDto {
+    @NotNull
+    @Email
     private String email;
+    @NotNull
+    @Size(min = 4, max = 255)
     private String password;
```

Comment:
not-null constraints can be removed from here since we duplicated it in our custom validators